notebooks/official/generative_ai/text_embedding_new_api.ipynb

```diff
@@ -0,0 +1,449 @@
+{
```

Comment:
<p>change See text embeddings api ... to Learn more about ....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done!</p>